src/features/bookings/helpers/expirationDateUtils.ts

```diff
@@ -0,0 +1,35 @@
+import { Booking } from 'features/bookings/types'
+
+export function getDigitalBookingWithoutExpirationDate(bookings: Booking[]) {
+  return bookings.filter((booking) => {
+    const isDigital = booking.stock.offer.isDigital
+    return isDigital === true && !booking.expirationDate
+  })
+}
+
+export function isBookingInList(
+  booking: Booking,
+  getDigitalBookingWithoutExpirationDate?: Booking[]
+) {
+  return getDigitalBookingWithoutExpirationDate?.some((b) => b.id === booking.id)
```

Comment:
```suggestion
  return getDigitalBookingWithoutExpirationDate?.some((b) => b.id === booking.id) ?? false
```

Ca force le type de retour